base_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'nameable'
+
+class BaseDecorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    @nameable = nameable
+    super()
```

Comment:
- [ ] **Optional:** A call to the `super()` isn't necessary.

Reply:
Thank you @brytebee